ForceTouchPlayer/Songs/Doom.swift

```diff
@@ -0,0 +1,134 @@
+//
+//  Doom.swift
+//  ForceTouchPlayer
+//
+//  Created by Henry Asbridge on 12/02/2021.
+//
+
+// Transcribed from the arduinosongs repo
+import Foundation
+
+let doomE1M1 = Song(
+    name: "Doom E1M1",
+    defaultTempo: 300.0,
+    rawNotes: [part1, part2, part3].flatMap { $0 }
```

Comment:
I played around a bit and reducing the padding timing from the default `0.25` to `0.1` better represented this song's tempo
```suggestion
let doomE1M1 = Song(
    name: "Doom E1M1",
    defaultTempo: 300.0,
    padding: Note(frequency: 0, value: 0.15),
    rawNotes: [part1, part2, part3].flatMap { $0 }
```